### PR TITLE
CON-206: populate validator performance doc

### PR DESCRIPTION
**Pull request by DannyS03**
- Adds framework content
- Follows parameters and calculations in original proposal (need to double-check).

### CODE REVIEW

File: docs/maintain/validate/validator-performance.md [hunk 1]
```diff
@@ -16,10 +16,142 @@ slug: validator-performance
...
+
+:::
+
+The framework allows for further decentralization of the network using validator self-regulation.
+As a result of the increased amount of validator churn that may occur due to the new on-chain validator management
+mechanism, the process for admitting new validators into the active set promotes network health and aligns with the notion
+of gradual decentralization. Increasing the validator set to 105 also increases the economic security and decentralization
+of the network.
+
+## Validator performance framework
```
Comment (ashucoder9) [inline docs/maintain/validate/validator-performance.md:54]:
```suggestion
## Validator Performance Framework
```

File: docs/maintain/validate/validator-performance.md [hunk 2]
```diff
@@ -16,10 +16,142 @@ slug: validator-performance
...
+:::note In a future release
+
+The dashboard will provide direct notifications to validators, alerting them to their
+deficient status.
+
+:::
+
+### Notice to Validators
+
+Validators are given 24 hours' notice of the commencement of performance monitoring, with the data set beginning 24 hours
```
Comment (ashucoder9) [inline docs/maintain/validate/validator-performance.md:91]:
```suggestion
Validators are given 24 hours notice of the commencement of performance monitoring, with the data set beginning 24 hours
```

File: docs/maintain/validate/validator-performance.md [hunk 3]
```diff
@@ -16,10 +16,142 @@ slug: validator-performance
...
+after the notice is sent.
+
+## Falling below performance benchmarks
```
Comment (ashucoder9) [inline docs/maintain/validate/validator-performance.md:94]:
```suggestion
## Falling below Performance Benchmarks
```

File: docs/maintain/validate/validator-performance.md [hunk 4]
```diff
@@ -16,10 +16,142 @@ slug: validator-performance
...
+forced exit procedure by offboarding the validator from the network by unbonding their stake.
 
 :::
 
-The Polygon Governance team administers an off-chain validator performance metric initative that assists validators in evaluating their network participation. Validators will eventually need to self-regulate network participation on Polygon to an agreed set of on-chain parameters as they network continues to decentralize. The validator performance metrics follow a benchmark calculation that evaluates validator performance.
+### Forced Unstaking
+
+The unstaking of the deficient validator would be done as follows:
+
+**Call ForceUnstake function in Polygon Commitchain Contract: 0xFa7D2a996aC6350f4b56C043112Da0366a59b74c**
```
Comment (ashucoder9) [inline docs/maintain/validate/validator-performance.md:118]:
```suggestion
**Call `ForceUnstake()` function in Polygon Commitchain Contract: 0xFa7D2a996aC6350f4b56C043112Da0366a59b74c**
```

File: docs/maintain/validate/validator-performance.md [hunk 5]
```diff
@@ -16,10 +16,142 @@ slug: validator-performance
...
+- Phase 1 (current) - The multi-sig holders would set the admission parameters and choose the validators.
+- Phase 2 (next step) - The governance and validator communities set and approve the parameters, and the multi-sig holders
+  implement those parameters.
+- Phase 3 (end state) - The governance and validator communities set the parameters and admit and onboard validators themselves.
+
+### Irregular joining
+
+Joining the validator network consists of two steps: **admission and onboarding**. These steps and specific parameters
+relating to each will ultimately be controlled by the governance community. This community approach enhances network security with
+a level of certainty about the intent of the actors trying to join the network, and introduces social ramifications, which can ensure
```
Comment (ashucoder9) [inline docs/maintain/validate/validator-performance.md:134]:
```suggestion
a level of certainty about the intent of the actors trying to join the network, and introduces social ramifications, which can ensure
```

**Review by ashucoder9 — APPROVED** [review]

**Comment by DannyS03** [comment]
continued in https://github.com/maticnetwork/matic-docs/pull/1239

**Pull request closed without merging**